### DISCUSSION TITLE
The "obtain all items" button is gone!!!

### DISCUSSION THREAD

**Discussion by ghost** [Prodigy-Hacking/ProdigyMathGameHacking#909] [(B) Bug Discussion]
pls fix this i hate using selector advanced

**Comment by LeoBadeaux** [comment]
Fix it yourself, after all you are the "BEST PRODIGY HACKER"

**Reply by NX721** [reply]
lmao

**Comment by ArcerionDev** [comment]
yeah alright I'll make a full stacker later
